File: app/src/main/java/spave/levan/waterever/ui/activities/SettingsActivity.java
package spave.levan.waterever.ui.activities;

import android.os.Bundle;
import android.support.annotation.Nullable;

import butterknife.ButterKnife;
import spave.levan.waterever.R;

/**
 * 佛祖保佑 永无BUG
 *
 * @author WangZhiYao
 * @date 2018/9/16
 */
public class SettingsActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
        ButterKnife.bind(this);

    }
}

File: app/src/main/kotlin/me/zhiyao/waterever/ui/settings/SettingsActivity.kt
package me.zhiyao.waterever.ui.settings

import android.os.Bundle
import me.zhiyao.waterever.databinding.ActivitySettingsBinding
import me.zhiyao.waterever.ui.base.BaseActivity

/**
 *
 * @author WangZhiYao
 * @date 2020/7/14
 */
class SettingsActivity : BaseActivity() {

    private lateinit var mBinding: ActivitySettingsBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mBinding = ActivitySettingsBinding.inflate(layoutInflater)
        setContentView(mBinding.root)
    }

    override fun showBack(): Boolean = true
}